controllers/limitador_status.go

```diff
@@ -108,6 +125,117 @@ func (r *LimitadorReconciler) readyCondition(ctx context.Context, limitadorObj *
 	return cond, nil
 }
 
+func (r *LimitadorReconciler) availableCondition(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) (*metav1.Condition, bool, error) {
+	cond := &metav1.Condition{
+		Type:    limitadorv1alpha1.StatusConditionAvailable,
+		Status:  metav1.ConditionTrue,
+		Reason:  "Available",
+		Message: "Limitador limits synced",
+	}
+
+	reason, err := r.checkLimitadorSynced(ctx, limitadorObj)
+	if err != nil {
+		return nil, true, err
+	}
+	if reason != nil {
+		cond.Status = metav1.ConditionFalse
+		cond.Reason = "LimitadorNotAvailable"
+		cond.Message = *reason
+		return cond, true, nil
+	}
+
+	return cond, false, nil
+}
+
+func (r *LimitadorReconciler) checkLimitadorSynced(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) (*string, error) {
+	// Create Kubernetes clientset.
+	clientset, err := kubernetes.NewForConfig(r.RestConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	podList := &corev1.PodList{}
+	options := &client.ListOptions{
+		LabelSelector: labels.SelectorFromSet(limitador.Labels(limitadorObj)),
+		Namespace:     limitadorObj.Namespace,
+	}
+	if err := r.Client().List(ctx, podList, options); err != nil {
+		return nil, err
+	}
+
+	if len(podList.Items) == 0 {
+		return nil, errors.New("found 0 pods")
+	}
+
+	// Name of the pod where the function will be executed.
+	podName := podList.Items[0].Name
+
+	// Command to execute your function.
+	command := []string{"cat", fmt.Sprintf("%s/%s", limitador.LimitadorCMMountPath, limitador.LimitadorConfigFileName)}
```

Comment:
good idea. 

If it wasn't for the requeue thing, I would buy it (*shut up and take my money* way).

Reply:
Yeah, endpoint to dump all the limits would be nice for comparison sake 👍